CHANGELOG.md

```diff
@@ -42,6 +42,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * Update the swagger files (including third-party changes). [#728](https://github.com/provenance-io/provenance/issues/728)
 * Bump IBC to 2.3.0 and update third-party protos [PR 868](https://github.com/provenance-io/provenance/pull/868)
 
+### Features
+
+* Bump Cosmos-SDK to `v0.45.4-pio-4` (from `v0.45.4-pio-2`) to utilize the new `CountAuthorization` authz grant type. [#807](https://github.com/provenance-io/provenance/issues/807)
+* Update metadata module authz handling to properly call `Accept` and delete/update authorizations as they're used [#728](https://github.com/provenance-io/provenance/issues/728)
```

Comment:
Issue 728 is for swagger

Reply:
I updated the title to reflect the change only for `metadata` module. 

No change required for the `marker` module. It already handles `accept.Delete`.
https://github.com/provenance-io/provenance/blob/a2aebd5971e06fbbb796dd2b4af1306524d8ec3a/x/marker/keeper/marker.go#L699-L717